pkg/registry/core/node/strategy.go

```diff
@@ -223,18 +223,12 @@ func GetAttrs(obj runtime.Object) (labels.Set, fields.Set, error) {
 // MatchNode returns a generic matcher for a given label and field selector.
 func MatchNode(label labels.Selector, field fields.Selector) pkgstorage.SelectionPredicate {
 	return pkgstorage.SelectionPredicate{
-		Label:       label,
-		Field:       field,
-		GetAttrs:    GetAttrs,
-		IndexFields: []string{"metadata.name"},
```

Comment:
I do not fully understand the changes to cacher but don't we still need this index?

Reply:
no, partitioning of watchers which are scoped to a specific namespace and/or name is automatic now, no special index required